weather-wise/pom.xml

```diff
@@ -15,6 +15,11 @@
     </properties>
 
     <dependencies>
+        <dependency>
+            <groupId>com.googlecode.json-simple</groupId>
+            <artifactId>json-simple</artifactId>
+            <version>1.1.1</version>
+        </dependency>
```

Comment:
Added dependencies for the JSON Simple library used for parsing and decoding JSON data. More documentation can be found [here](https://mvnrepository.com/artifact/com.googlecode.json-simple/json-simple/1.1.1).